diff --git a/build.zig b/build.zig
index 09cc709..d4ca253 100644
--- a/build.zig
+++ b/build.zig
@@ -7,9 +7,9 @@ pub fn build(b: *Builder) void {
     exe.setBuildMode(mode);
 
     // on normal systems this is ok:
-    //exe.linkSystemLibrary("sodium");
+    exe.linkSystemLibrary("sodium");
     // on debian, you have to do this:
-    exe.addObjectFile("/usr/lib/x86_64-linux-gnu/libsodium.a");
+    // exe.addObjectFile("/usr/lib/x86_64-linux-gnu/libsodium.a");
 
     exe.linkLibC();
 
